### PR TITLE
Update troubleshoot-microsoft-store-for-business.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
added link to the troubleshooting guide, issue #1664

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: store-for-business/troubleshoot-microsoft-store-for-business.md [hunk 1]
```diff
@@ -49,11 +49,15 @@ The private store for your organization is a page in Microsoft Store app that co
 
     ![Private store for Contoso publishing](images/wsfb-privatestoreapps.png)
 
+## Troubleshooting Microsoft Store for Business integration with ConfigMgr
+
+In case of having problems with Microsoft Store for Business integration with ConfigMgr, use the [troubleshooting guide](https://support.microsoft.com/help/4010214/understand-and-troubleshoot-microsoft-store-for-business-integration-w).
```
Comment (JohanFreelancer9) [inline store-for-business/troubleshoot-microsoft-store-for-business.md]:
```suggestion
If you encounter any problems when integrating Microsoft Store for Business with ConfigMgr, use the [troubleshooting guide](https://support.microsoft.com/help/4010214/understand-and-troubleshoot-microsoft-store-for-business-integration-w).
```

File: store-for-business/troubleshoot-microsoft-store-for-business.md [hunk 2]
```diff
@@ -49,11 +49,15 @@ The private store for your organization is a page in Microsoft Store app that co
...
 ## Still having trouble?
 
 If you are still having trouble using Microsoft Store or installing an app, Admins can sign in and look for topics on our **Support** page.
    
 **To view Support page** 
 
 1. Sign in to [Microsoft Store for Business](https://businessstore.microsoft.com)
-2. Click **Manage**, and then click **Support**. 
\ No newline at end of file
+2. Click **Manage**, and then click **Support**.
```
Comment (JohanFreelancer9) [inline store-for-business/troubleshoot-microsoft-store-for-business.md]:
```suggestion
2. Choose **Manage**> **Support**. 
```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were applied, thank you!

**Comment by mypil** [comment]
@TrudyHa @jaimeo @lizap - Can you please assist to check, approve and merge the content of this PR based on issue #1664?

Thank you.

**Comment by lizap** [comment]
@mypil et al - I've reviewed and will merge now. However, please note that I had to correct the reference to Configuration Manager to use the actual product name. Please use caution when referring to Microsoft services and products - no nicknames or abbreviations, please.

**Comment by mypil** [comment]
@lizap - this is noted. Thank you for your help in merging the PR content. Looping in @nenonix so this info could be added in the Wiki.

**Pull request merged**